Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -177,58 +173,69 @@ fun DetailsContent(
 }
 
 @Composable
-private fun CityMapView(latitude: String, longitude: String) {
-    // The MapView lifecycle is handled by this composable. As the MapView also needs to be updated
-    // with input from Compose UI, those updates are encapsulated into the MapViewContainer
-    // composable. In this way, when an update to the MapView happens, this composable won't
-    // recompose and the MapView won't need to be recreated.
-    val mapView = rememberMapViewWithLifecycle()
-    MapViewContainer(mapView, latitude, longitude)
-}
-
-@Composable
-private fun MapViewContainer(
-    map: MapView,
+fun CityMapView(
     latitude: String,
-    longitude: String
+    longitude: String,
+    cameraState: CameraPositionState = rememberSaveable(saver = CameraPositionState.Saver) { CameraPositionState() },
```

Comment:
```suggestion
    cameraPositionState: CameraPositionState = rememberSaveable(saver = CameraPositionState.Saver) { CameraPositionState() },
```

Are there any other camera states? might be better to be explicit in the name to avoid confusion